dev/conductor/ui/test/widget_test.dart

```diff
@@ -6,17 +6,17 @@ import 'dart:io' show Platform;
 
 import 'package:conductor_core/proto.dart' as pb;
 import 'package:conductor_ui/main.dart';
```

Comment:
nit: rename the file main_test.dart

Tests should match to where the relevant code is